espanol/muralla/abrir/abrirsoldados/cerrad/rocas/rocas.md

```diff
@@ -1,7 +1,9 @@
-�Los soldados enemigos eran demasiados! 
+¡Los soldados enemigos eran demasiados! 
 
-Tomaron la decisi�n de bloquear la puerta con rocas, troncos y todos los objetos pesados que ten�an a mano.
+Tomaron la decisión de bloquear la puerta con rocas, troncos y todos los objetos pesados que tenían a mano.
 
-Al menos por el momento el ej�rcito enemigo no pod�a entrar derribando la puerta con un ariete.
+Al menos por el momento el ejército enemigo no podía entrar derribando la puerta con un ariete.
 
-Ahora ten�an tiempo para apostar arqueros en los muros y organizar la defensa.
+Ahora tenían tiempo para apostar arqueros en los muros y organizar la defensa.
+
+Pero no sabían qué hacer, así que lo buscaron en [google](www.google.com)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.